styles.css

```diff
@@ -0,0 +1,58 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+.heading-section {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] According to the Figma template, the section of this project has a background image but that seems to be missing in your project. Kindly follow this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=kgjXcDYS4nMWzLhZ-0) of the Figma template to download the proper image and use.
- Click this [link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) to learn more about CSS-background image